src/models/Settings.js

```diff
@@ -46,6 +46,7 @@ function (Okta, Errors, BrowserFeatures, Util, Logger, config) {
       'recoveryToken': ['string', false, undefined],
       'stateToken': ['string', false, undefined],
       'username' : ['string', false],
+      'logoText' : ['string', false],
```

Comment:
We need `logo` (contains path to image) and `logoText` (will be org.name for the alt attribute).